exporters/jaeger/include/opentelemetry/exporters/jaeger/jaeger_exporter_options.h

```diff
@@ -21,7 +27,7 @@ enum class TransportFormat
 /**
  * Struct to hold Jaeger exporter options.
  */
-struct JaegerExporterOptions
+struct OPENTELEMETRY_DEPRECATED JaegerExporterOptions
```

Comment:
Could we just show warning when including the header file, or we want to show separate message for using each construct in the header file?

Reply:
Both will work.

In this case, the file exposes only one class, so it is really equivalent.